com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/steps/CreateAppStepTest.java

```diff
@@ -235,4 +242,140 @@ protected CreateAppStep createStep() {
         return new CreateAppStep();
     }
 
+    public static class TestWithDocker extends SyncFlowableStepTest<CreateAppStep> {
```

Comment:
I just noticed that a lot of the methods in this class are copy-pasted from the test above. While some duplication is fine in the tests, copy-pasting 50 lines of code is a bit too much. ;) In the future, try to reuse such large blocks of code.